src/Parser/Core/Modules/Items/ArchimondesHatredReborn.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+
+import Module from 'Parser/Core/Module';
+
+class ArchimondesHatredReborn extends Module {
+  healing = 0;
+
+  on_initialized() {
+    if (!this.owner.error) {
+      this.active = this.owner.selectedCombatant.hasTrinket(ITEMS.ARCHIMONDES_HATRED_REBORN.id);
+    }
+  }
+
+  on_byPlayer_absorbed(event) {
+    const spellId = event.ability.guid;
+    if (spellId === ITEMS.ARCHIMONDES_HATRED_REBORN.absorbId) {
+      this.healing += event.amount;
+    }
+  }
+
+  item() {
+    return {
+      item: ITEMS.ARCHIMONDES_HATRED_REBORN,
+      result: (
+        <dfn data-tip={`The total damaged absorbed by Archimonde's Hatred Reborn was ${this.owner.formatItemAbsorbDone(this.healing)}.`} >
```

Comment:
For these tanking legendaries, it would be great to have the total amount of absorbed damage. At least for me, the Effective Healing / HPS on a shield item doesn't mean much to me (as a tank) and it can be quite confusing of those meters. But if you really think this tooltips should be removed, I can do it asap.